src/api/search/src/routes/query.js

```diff
@@ -12,4 +12,13 @@ router.get('/', validateQuery, async (req, res, next) => {
   }
 });
 
+// query for author autocomplete
+router.get('/authors/autocomplete', validateQuery, async (req, res, next) => {
```

Comment:
I wonder if we need to write another validator, since `validateQuery` is overly broad for what you need here.  That would solve your 2 character limit too.  What about adding `validateAuthor` in https://github.com/Seneca-CDOT/telescope/blob/d8867e9c7f001574ec85630103653dc92a93acfc/src/api/search/src/bin/validation.js.